NRzure/quickstarts/example-quickstart/config.yml

```diff
@@ -0,0 +1,51 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: AzureAD
+
+# Displayed in the UI (required)
+title: Azure Active Directory
+
+# Long-form description of the quickstart (required)
+description: |
+  This quickstart allows for basic data collection about information related to the Azure Active Directory services.
```

Comment:
Let's fill this out with a bit more description of the data rendered in this dashboard, also if alerts are to be included they should be mentioned here as well.